### PR TITLE
feat(header): header component. basic skeleton for header component, …

### LINKED ISSUE #7: Header
Create a header component that support features we need locally for TLP.

**Tasks:**
- [ ] #8 
- [ ] #9 
- [ ] #10 

Image

### PR DESCRIPTION
**Pull request by jensmo98**
resolves #7

-Create basic layout for header.
-Make it mobile friendly and responsiv

### CODE REVIEW

File: packages/react/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -41,27 +40,110 @@ interface HeaderBottomProps {
...
+  const handleResize = () => {
+    if (window.innerWidth < breakpoint) {
+      setIsMobile(true);
+    } else {
+      setIsMobile(false);
+    }
+  };
+
+  useEffect(() => {
+    // Kjører en gang ved oppstart for å sette riktig verdifor isMobile
```
Comment (Thuneer) [inline packages/react/src/components/Header/Header.tsx:61]: Try to use English for all code and comments.

**Comment by Thuneer** [comment]
Just a comment to make sure you have thought about this: When a user clicks on a Link inside the mobile menu, the menu should close. There could be other actions inside the mobile menu that might require the menu to close also. I suggest adding a `closeMenu` event prop so that the end user can determine for themselves when the mobile menu should close or not.

File: packages/react/src/components/Header/Header.stories.tsx [hunk 2]
```diff
@@ -1,11 +1,154 @@
...
+  <Header.Left key={0}>
+    <a href='.'>
+      <img
+        src={myLogo}
+        alt='SVG logo'
+      />
+    </a>
+  </Header.Left>,
+  <Header.Middle key={1}>
+    <nav>
```
Comment (Thuneer) [inline packages/react/src/components/Header/Header.stories.tsx:19]: The HTML inside the header regions for the story seem to repeat themselves a lot (DRY). Maybe try to make the code more reusable.

File: packages/react/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -41,27 +40,110 @@ interface HeaderBottomProps {
...
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const toggleMenu = () => {
+    setShowMenu(!showMenu);
```
[the remark this answers is not in the document]
Comment (jensmo98) [reply packages/react/src/components/Header/Header.tsx:73]: Not sure what you mean with update function? !isMenuOpen is always opposite of current value..?

[the remark this answers is not in the document]
Comment (Thuneer) [reply packages/react/src/components/Header/Header.tsx:73]: Read about it here, is is best practice to use it when updating a value based on previous state: https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state

**Review by Thuneer — APPROVED** [review]
LGTM 👍

**Pull request merged**